span.go

```diff
@@ -112,13 +112,12 @@ func (s *Span) SetTag(key string, value interface{}) opentracing.Span {
 }
 
 func (s *Span) setTagNoLocking(key string, value interface{}) {
-	handled := false
 	if handler, ok := specialTagHandlers[key]; ok {
-		handled = handler(s, key, value)
-	}
-	if !handled {
-		s.tags = append(s.tags, Tag{key: key, value: value})
+		if !handler(s, key, value) {
```

Comment:
I think once we switch to jaeger.thrift, the only special handler we need is for sampling.priority